tsconfig.json

```diff
@@ -0,0 +1,32 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```

Comment:
I would add `"outDir": `./build`.   (We plan to add that for Vega-Lite pretty soon too.)